resources/knative-kafka/values.yaml

```diff
@@ -0,0 +1,67 @@
+# Default values for kafka.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Knative Requires that the Deployment be in the knative-eventing namespace (this in turn requires that the ServiceAccount must also be there)
+# The Runtime Namespace is where the Channel/Dispatcher Deployments/Services will be created (and is expected to be where the channels lives)
+global:
```

Comment:
The runtime namespace was removed but the comment above is still in place?  Not necessarily an issue for this PR but we maybe need to think about expanding from our single namespace approach and catch up with Knative namespace handling some?